Documentation/Books/Manual/Views/ArangoSearch/README.md

```diff
@@ -1,54 +1,293 @@
+* sort the result set based on how closely each document matched the search condition
 
 A concept of value 'analysis' that is meant to break up a given value into
 a set of sub-values internally tied together by metadata which influences both
-the filter and sort stages to provide the most appropriate match for the
+the search and sort stages to provide the most appropriate match for the
 specified conditions, similar to queries to web search engines.
 
 In plain terms this means a user can for example:
-
 * request documents where the 'body' attribute best matches 'a quick brown fox'
 * request documents where the 'dna' attribute best matches a DNA sub sequence
 * request documents where the 'name' attribute best matches gender
-* etc. (via custom analyzers)
+* etc... (via custom analyzers described in the next section)
 
-## The IResearch Library
+### The IResearch Library
 
-IResearch is a cross-platform open source indexing and searching engine written
-in modern C++, optimized for speed and memory footprint, with source available
-from https://github.com/iresearch-toolkit/iresearch
+IResearch s a cross-platform open source indexing and searching engine written in C++,
+optimized for speed and memory footprint, with source available from:
+https://github.com/iresearch-toolkit/iresearch
 
-IResearch is the framework for indexing, filtering and sorting of data.
-The indexing stage can treat each data item as an atom or use custom 'analyzers'
-to break the data item into sub-atomic pieces tied together with internally
-tracked metadata.
+IResearch is a framework for indexing, searching and sorting of data. The indexing stage can
+treat each data item as an atom or use custom 'analyzers' to break the data item
+into sub-atomic pieces tied together with internally tracked metadata.
 
 The IResearch framework in general can be further extended at runtime with
-custom implementations of analyzers (used during the indexing and filtering
+custom implementations of analyzers (used during the indexing and searching
 stages) and scorers (used during the sorting stage) allowing full control over
 the behavior of the engine.
 
-## Using ArangoSearch views
-
-To get more familiar with ArangoSearch usage, you may start with [Getting Started](GettingStarted.md) simple guide and then explore details of ArangoSearch in
- [Detailed Overview](DetailedOverview.md),
- [Analyzers](Analyzers.md)
- and [Scorers](Scorers.md) topics.
-    currently supported types are:
-    - *bytes*:
-      Consolidate IFF {threshold} > segment_bytes / (all_segment_bytes / #segments)
-    - *bytes_accum*:
-      Consolidate IFF {threshold} > (segment_bytes + sum_of_merge_candidate_segment_bytes) / all_segment_bytes
-    - *count*:
-      Consolidate IFF {threshold} > segment_docs{valid} / (all_segment_docs{valid} / #segments)
-    - *fill*:
-      Consolidate IFF {threshold} > #segment_docs{valid} / (#segment_docs{valid} + #segment_docs{removed})
+
+### ArangoSearch Scorers
```

Comment:
Is this reverting all changes just made in https://github.com/arangodb/arangodb/pull/6130 ?!

Reply:
I've not seen that pull-request, I've only added the clarifications requested by @neunhoef and created this pull request

Reply:
@KVS85 Can you help to sort this out? I'm afraid this will be tricky and involve intellectual merging of content. Maybe do `git diff cb12be3e...HEAD` while this PR is checked out? (the commit hash is the commit before #6130 was merged into devel). Then you should see better what changes were made in this PR, and then somehow combine them with the changes made in #6130.